ctl/main.go

```diff
@@ -43,6 +44,7 @@ func main() {
 	rootCmd.AddCommand(waypoint.NewCmd())
 	rootCmd.AddCommand(version.NewCmd())
 	rootCmd.AddCommand(accesslog.NewCmd())
+	rootCmd.AddCommand(install.NewCmd())
```

Comment:
Also need to update [doc](https://github.com/kmesh-net/website/blob/main/content/en/docs/userguide/use_kmeshctl.md) when this PR get merged.